mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -102,6 +102,7 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_25296 LOOT_FUNCTION_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_25297 LOOT_CONDITION_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_25299 LOOT_CONDITION_TYPE Lnet/minecraft/class_2378;
+	FIELD field_25490 DIMENSION_DATA Lnet/minecraft/class_5321;
```

Comment:
Wonder about this name... a combination of chunk generator and dimension type

Reply:
What about DimensionOptions?

Reply:
that actually makes sense as that is the sub option in generator options :+1: